entity-framework/core/get-started/uwp/getting-started.md

```diff
@@ -170,7 +167,7 @@ You can now use your model to perform data access.
 * Add the page load handler and UI content highlighted below
 
 <!-- [!code-csharp[Main](samples/core/GetStarted/UWP/UWP.SQLite/MainPage.xaml?highlight=9,12,13,14,15,16,17,18,19,20,21,22)] -->
-````csharp
+``` xaml
```

Comment:
It seems to work. I don't know that it does anything special that XML wouldn't do, but it shows it is XAML in the title of the snippet :smile: